File: 1528-shuffle-string/1528-shuffle-string.cpp
class Solution {
public:
    string restoreString(string s, vector<int>& indices) {
       string ans ="";

        for(int i = 0; i<s.length();i++){
          for(int j = 0; j < indices.size();j++){
              if(i == indices[j]){
                  ans += s[j];
              }
          }            
        }
        return ans;
    }
};

File: 1528-shuffle-string/1528-shuffle-string.py
class Solution:
    def restoreString(self, s: str, indices: List[int]) -> str:
        str1=''
        for i in range(len(s)):
            for j in range(len(indices)):
                if i==indices[j]:
                    str1=str1+s[j]
        return str1